go-controller/pkg/ovn/gateway_test.go

```diff
@@ -114,6 +114,16 @@ func generateGatewayInitExpectedNB(testData []libovsdb.TestData, expectedOVNClus
 		},
 		Networks: networks,
 	})
+	natUUID := fmt.Sprintf("router-snat-%v-UUID", GRName)
```

Comment:
@alexanderConstantinescu I'm missing something important here! Please help.

Reply:
Without knowing the error you're getting, I'd say: use `libovsdbops.GenerateNamedUUID()` for this named UUID.